core/build.gradle.kts

```diff
@@ -35,6 +42,7 @@ kotlin {
         val androidMain by getting {
             dependencies {
                 implementation(libs.coroutinesAndroid)
+                implementation(libs.ktor.okhttp)
```

Comment:
unused dependency?

Reply:
Nope, otherwise ktor fails to configure the engine:       
> IllegalStateException: Failed to find HTTP client engine implementation in the classpath: consider adding client engine dependency.